lib/bridge/MatrixHandler.js

```diff
@@ -770,6 +769,68 @@ MatrixHandler.prototype._onMessage = Promise.coroutine(function*(req, event) {
     yield Promise.all(promises);
 });
 
+MatrixHandler.prototype._sendIrcAction = Promise.coroutine(
+    function*(req, ircRoom, ircUser, ircAction, event) {
```

Comment:
I know you're just using the same named variables as before, but the name `ircUser` is terrible - that variable *isn't* an IRC user, it's a `BridgedClient`. \*chastises past self\* - please rename this to `client` or `ircClient` or something like that.